### PR TITLE
Update User Guide for user profile

### LINKED ISSUE #225: Add List and Find features to the User Guide

### LINKED ISSUE #230: Add images for profile commands - user guide
- Add, delete, edit, view

### LINKED ISSUE #235: Add find and list command profile - user guide

### PR DESCRIPTION
**Pull request by orzymandias**
Summary of PR
* Update profile segment of UG with find and list commands
* Update UG with images to supplement commands

closes #235 
closes #230 
closes #225

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ title: User Guide
   [3.4.2. Deleting item in profile: `delete me`](#deleting-item-in-profile-delete-me) <br />
   [3.4.3. Editing item in profile: `edit me`](#editing-item-in-profile-edit-me) <br />
   [3.4.4. Viewing item in profile: `view me`](#viewing-item-in-profile-view-me) <br />
+  [3.1.5. Finding items in profile: `find me`](#finding-items-in-profile-find-me) <br />
```
Comment (seanjyjy) [inline docs/UserGuide.md]: the numbering is wrong, but I have updated this in my current PR for its ToC

File: docs/UserGuide.md [hunk 2]
```diff
@@ -326,32 +329,35 @@ Example:
...
 
 #### Adding item to profile: `add me`
 
-Adds experience, skills or achievements descriptors to your profile.
+Adds a profile item to your profile.
 
 Format: `add me c/CATEGORY t/TITLE [d/DESCRIPTOR]...`
 
-* Category specifies a category which can be either experience, skills or achievement.
+* `CATEGORY` specifies a category which can be either experience, skills or achievement.
```
Comment (seanjyjy) [inline docs/UserGuide.md:433]: not sure if this is a repetition of line 333 to 336. If it isn't you might want to encase experience skills achievements with the `experience` individually

File: docs/UserGuide.md [hunk 3]
```diff
@@ -326,32 +329,35 @@ Example:
...
 Examples:
-* `add me c/experience t/Internship at Govtech d/Implemented automate testing 
-using TravisCI d/Implemented dashboard to track code coverage`
-* `add me c/achievement t/special recognition in Hack n Roll`
 * `add me c/skill t/HTML d/Learn how to create divs`
-
+* `add me c/achievement t/special recognition in Hack n Roll`
+* For example, executing `add me c/experience t/Internship at Govtech d/Implemented automate testing 
+using TravisCI d/Implemented dashboard to track code coverage` on an empty profile will add the profile item as follows: <br />
+![AddProfileSS](images/AddProfileItemSS.PNG)
```
Comment (seanjyjy) [inline docs/UserGuide.md]: it should be .png, I think images are case-sensitive.

**Review by seanjyjy — COMMENTED** [review]
LGTM just some nits to fix

File: docs/UserGuide.md (hunk 1 above)
Comment (orzymandias) [reply docs/UserGuide.md]: Okay but my find header is different, I had a different header from shawn but he hasnt gotten back to me about changing yet

File: docs/UserGuide.md (hunk 2 above)
Comment (orzymandias) [reply docs/UserGuide.md:433]: Line 333 is a brief description of profile items and 336 is a more formal definition. I think its mainly for the overview in  2101.

File: docs/UserGuide.md (hunk 3 above)
Comment (orzymandias) [reply docs/UserGuide.md]: The images turn out well in my markdown on intellij. Also the images in company all have this casing too

Comment (seanjyjy) [reply docs/UserGuide.md]: oh i think intellij doesnt care about case-sensitive but github does haha

**Review by shawn-nyk — APPROVED** [review]
LGTM

**Pull request merged**